packages/rancher-tracing/charts/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.20.0
 description: A quick start Jaeger Tracing installation using the all-in-one demo. This is not production qualified. Refer to https://www.jaegertracing.io/ for details.
 name: rancher-tracing
-version: 1.20.0
+version: 1.20.1
```

Comment:
So both versions will use same kiali and jaeger ?

Reply:
Which is fine, just wanted to confirm

Reply:
Yes, since both 1.8 and 1.9 can use versions of kaili greater than 1.26, and jaeger can usually be the latest for all version of istio as long as there are deprecations on the zipkin api.